client/src/Pages/Register/index.js

```diff
@@ -0,0 +1,170 @@
+    try {
+      setLoading(true);
+      setError('');
+      await Axios.post('/api/v1/signup', userData);
+      setLoading(false);
+      setIsAuth(true);
+    } catch (err) {
+      if (err.response) {
+        setError(err.response.data.message || 'Something went wrong!');
+      }
+      setLoading(false);
+    }
+  };
+
+  return (
+    <Row className="signup">
+      <Col className="left-section" span={9}>
+        <Title className="title" level={2}>
+          Welcame Back
+        </Title>
+        <Paragraph className="desc">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+        </Paragraph>
+        <Link className="signin-link" to={LOGIN_PAGE}>
+          Signin
+        </Link>
+      </Col>
+      <Col span={15} className="signup-form-container">
+        <Title className="title" level={2}>
+          Create A New Account
+        </Title>
+        {error && <Alert id="alert" message={error} type="info" showIcon />}
+        <Form
+          onFinish={onFinish}
+          initialValues={{
+            role: 'user',
+            location: 'gaza',
+          }}
+        >
+          <Form.Item
+            label="Name"
+            name="username"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your Name!',
+              },
+              {
+                pattern: /^[a-z]([-']?[a-z]+)*( [a-z]([-']?[a-z]+)*)+$/,
+                message: 'Type your fullname.',
+              },
+            ]}
+          >
+            <Input placeholder="Enter your name..." />
+          </Form.Item>
+          <Form.Item
+            label="Email"
+            name="email"
+            rules={[
+              {
+                type: 'email',
+                message: 'Please enter a valid email!',
+              },
+              {
+                required: true,
+                message: 'Please input your Email!',
+              },
+            ]}
+          >
+            <Input placeholder="Enter a valid email..." />
+          </Form.Item>
+          <Form.Item
+            label="Mobile"
+            name="mobile"
+            rules={[
+              {
+                required: true,
+                message: 'Please input mobile number!',
+              },
+            ]}
+          >
+            <Input placeholder="Enter your mobile number..." />
+          </Form.Item>
+          <Form.Item
+            label="Password"
+            name="password"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your password!',
+              },
+            ]}
+          >
+            <Input type="password" placeholder="Enter password..." />
+          </Form.Item>
+          <Form.Item
+            label="Confirm Password"
+            name="confirmPassword"
+            rules={[
+              {
+                required: true,
+                message: 'Please input your the same password!',
```

Comment:
Please change the message to 'Please input the same password as above' for more text clarity.